lib/seahorn/BvOpSem2ExtraWideMemMgr.cc

```diff
@@ -542,6 +573,49 @@ template <class T>
 Expr ExtraWideMemManager<T>::bytesToSlotExpr(unsigned int bytes) {
   return m_ctx.alu().si(bytes, g_slotBitWidth);
 }
+template <class T>
+typename ExtraWideMemManager<T>::MemValTy
+ExtraWideMemManager<T>::memsetMetaData(ExtraWideMemManager::PtrTy ptr,
+                                       unsigned int len,
+                                       ExtraWideMemManager::MemValTy memIn,
+                                       uint32_t align, unsigned int val) {
+  RawMemValTy rawOut = memIn.getRaw();
+  if (HasTracking<T>::value) {
```

Comment:
this can be moved into method template parameter so that you have
`memsetMetaData<true>()` and `memsetMetaData<false>()`